src/main/java/seedu/address/storage/adapter/XmlAdaptedUser.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.storage.adapter;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import seedu.address.model.user.User;
+
+/**
+ * This represents a user in Trajectory.
+ */
+@XmlRootElement(name = "user")
+
+public class XmlAdaptedUser {
+
+    private String email;
+    private String unhashedPassword;
+    private String role;
+
+    public XmlAdaptedUser(){
```

Comment:
I'm surprised that the lack of spacing here (between the parentheses and the opening brace) got past checkstyles...